### ISSUE TITLE
Runtime errors from ply parsing code?

### ISSUE THREAD

**Issue by fredcy** [murbard/pytezos#7]
When I run an application that imports the pytezos library I get the following errors each time it starts up:

```
WARNING: No t_error rule is defined
WARNING: no p_error() function is defined
WARNING: Token 'COMMENT' defined, but not used
WARNING: Token 'MULTI_COMMENT' defined, but not used
WARNING: There are 2 unused tokens
```

FWIW, I have the python `logging` module enabled for my app with `logger.setLevel(logging.INFO)` in the global context.

I suspect that the messages arise from the use of the `ply` module in pytezos/micheline/grammar.py.

**Comment by m-kus** [comment]
Yep, this comes from `ply` module, will fix in the next release, thanks!
